admin_guide/install/prerequisites.adoc

```diff
@@ -216,8 +216,15 @@ endif::[]
 . Install the following packages:
 +
 ----
-# yum install wget git net-tools bind-utils iptables-services bridge-utils python-virtualenv
+# yum install wget git net-tools bind-utils iptables-services bridge-utils
 ----
++
+ifdef::openshift-enterprise[]
+If you are using the link:../../admin_guide/install/quick_install.html[Quick Installation Method] you must also install the GNU Compiler Collection (GCC) and the Python Virtual Environment (python-virtualenv) packages:
```

Comment:
Minor nits:

- The link works as-is, but just FYI you could simplify it to `link:quick_install.html` since this `prerequisites` topic is already in the `admin_guide/install/` directory.

- s/GCC/gcc (to match the other parenthetical which is showing the actual name of the package)

- Markup both "gcc" and "python-virtualenv" in the sentence as software packages per https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#quick-markup-reference.